docs/source/monitoring.md

```diff
@@ -1,76 +1,130 @@
-
-   Now the dashboards can be created like this:
-   ```
-   # Scylla dashboards
-   for f in scylla-monitoring/grafana/build/ver_4.6/*.json; do
-     kubectl -n scylla-monitoring create configmap scylla-dashboard-"$( basename "${f}" '.json' )" --from-file="${f}" --dry-run=client -o yaml | kubectl label -f- --dry-run=client -o yaml --local grafana_dashboard=1 | kubectl apply --server-side -f-
-   done
-
-   # Scylla Manager dashboards
-   for f in scylla-monitoring/grafana/build/manager_3.0/*.json; do
-     kubectl -n scylla-monitoring create configmap scylla-manager-dashboard-"$( basename "${f}" '.json' )" --from-file="${f}" --dry-run=client -o yaml | kubectl label -f- --dry-run=client -o yaml --local grafana_dashboard=1 | kubectl apply --server-side -f-
-   done
-    ```
-
-    Once Grafana sidecar picks up these dashboards they should be accessible in Grafana.
-
-To access Grafana locally, run:
-    ```
-    kubectl -n scylla-monitoring port-forward deployment.apps/monitoring-grafana 3000
-    ```
-
-   You can find it on `http://127.0.0.1:3000` and login with the credentials `admin`:`admin`.
\ No newline at end of file
+# Monitoring
+
+Scylla Operator 1.8 introduced a new API object `ScyllaDBMonitoring`, allowing users to deploy a managed monitoring 
+setup for their Scylla Clusters.
+
+```yaml
+apiVersion: scylla.scylladb.com/v1alpha1
+kind: ScyllaDBMonitoring
+metadata:
+  name: example
+spec:
+  type: Platform
+  endpointsSelector:
+    matchLabels:
+      app.kubernetes.io/name: scylla
+      scylla-operator.scylladb.com/scylla-service-type: identity
+      scylla/cluster: replace-with-your-scyllacluster-name
+  components:
+    prometheus:
+      storage:
+        volumeClaimTemplate:
+          spec:
+            resources:
+              requests:
+                storage: 1Gi
+    grafana:
+      exposeOptions:
+        webInterface:
+          ingress:
+            ingressClassName: haproxy
+            dnsDomains:
+            - test-grafana.test.svc.cluster.local
+            annotations:
+              haproxy-ingress.github.io/ssl-passthrough: "true"
+```
+
+For details, refer to the below command:
+```console
+$ kubectl explain scylladbmonitorings.scylla.scylladb.com/v1alpha1
+```
+
+## Deploy managed monitoring
+
+**Note**: as of v1.8, ScyllaDBMonitoring is in alpha state and is susceptible to backwards incompatible changes.
+
+### Requirements
+
+Before you can set up your ScyllaDB Monitoring, you must install the Scylla Operator in your Kubernetes cluster 
+and have an instance of ScyllaCluster running. For details see:
+* [Deploying Scylla on a Kubernetes Cluster](generic.md)
+* [Deploying Scylla stack using Helm Charts](helm.md)
+
+The managed monitoring setup also requires HAProxy Ingress and Prometheus Operator deployed in your Kubernetes cluster.
+You can deploy them using the provided third party examples.
+
+#### Deploy Prometheus Operator
+Deploy Prometheus Operator using kubectl:
```

Comment:
if your cluster already has a prometheus operator deployed you should skip this step

Reply:
I've put a message above saying you can skip these steps if you already have them deployed in your cluster.